Update README.md (#1)
HappyFerret/tennis-test: README.md

# Tennis-test

## Goal 

Create a scoring system for a game of tennis. Please note a "game" refers to the game component in a tennis match only. Theres is no need to cater for sets/matches. There is also no need for any form of GUI, just a simple class or collection of functions to create and maintain game state will suffice.

*Please note:* this is a pair programming excercise, so please don't hesistate to communicate and collaborate with the developers you are working with. Also feel free to use any resources online you might need. 

## The Task

Your solution should implement the following features:

- A game can be created with 2 players
- Each players score starts at 0
- Each player can score a point
- The score of a player in the game increments the following way: 0 - 15 - 30 - 40

> - The score resets to 0-0 when one player reaches 40.

Reviewer: This is a confusing way of saying someone wins the game.  
```suggestion
- A player wins the game if they score after reaching 40, and the match is not at Duece.
```

- Deuce can be reached if players score is tied at 40-40
- Advantage point will be awarded if a player scores at 40-40
- The game returns to deuce if one player is at advantage and the other scores a point

> - The score resets to 0-0 if one player scores twice in a row at Deuce

Reviewer: Again, the scenario is a win...
```suggestion
- If one player scores twice in a row at Deuce then they win the game
```

> 

Reviewer 2: ```suggestion
- Does anyone actually read this far?
```
